### PR TITLE
Add basic unit tests for Pipeline and Situp

### PR DESCRIPTION
**Pull request by yadavcbala**
*Issue #, if available:*

*Description of changes:*
* Add basic unit tests for Pipeline and Situp, will add more in subsequent PRs.
* Also update PluginSetting to avoid ClassCastException when getLong is called for int value attribute.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: situp-core/src/main/resources/situp-default.yml [hunk 1]
```diff
@@ -2,15 +2,17 @@ otel-es-raw-pipeline:
...
   delay: 3000
 otel-es-enrich-pipeline:
   source:
     pipeline:
       name: "otel-es-raw-pipeline"
   sink:
-    elasticsearch:
-      hosts: ["https://search-sample-app-test-lqwynrnd2ikcuzfsrdilv4stbq.us-west-2.es.amazonaws.com"]
+    - elasticsearch:
+        hosts: ["https://search-sample-app-test-lqwynrnd2ikcuzfsrdilv4stbq.us-west-2.es.amazonaws.com"]
```
Comment (laneholloway) [inline situp-core/src/main/resources/situp-default.yml:15]: This really should not be in the default yaml.

**Review by laneholloway — CHANGES_REQUESTED** [review]
Can we remove the default es cluster value and make it something obvious like `DEFAULT_CLUSTER_NOT_SET` so the user knows that it needs to be changed; or that they need to point to their own configuration file?

File: situp-core/src/main/resources/situp-default.yml (hunk 1 above)
Comment (kowshikn) [reply situp-core/src/main/resources/situp-default.yml:15]: Why should situp have a default yaml? This doesn't make sense, we should probable avoid this and ask users to always provide the yml fils as an argument.

Comment (yadavcbala) [reply situp-core/src/main/resources/situp-default.yml:15]: I agree with you but would prefer it have that change in its separate PR.

**Review by laneholloway — APPROVED** [review]
Approving so your next commit can remove the default file completely.

**Review by kowshikn — APPROVED** [review]

**Pull request merged**